unit-tests/src/test/scala/junit/JUnitExampleTest.scala

```diff
@@ -0,0 +1,10 @@
+package junit
+
+import org.junit.Assert._
+import org.junit.Test
+
+class JUnitExampleTest {
+  @Test def exampleTest(): Unit = {
+    assertTrue("This assertion should pass", condition = true)
```

Comment:
```suggestion
    assertTrue("This assertion should pass", true)
```
(the parameter name is not part of the API, and should not be referred to)